Tools/WebKitTestRunner/cocoa/WKTextExtractionTestingHelpers.mm

```diff
@@ -61,8 +62,15 @@ ASCIILiteral description(WKTextExtractionContainer container)
     }
 }
 
-static void buildDescriptionIgnoringChildren(TextStream& stream, WKTextExtractionItem *item)
+static void buildDescriptionIgnoringChildren(TextStream& stream, WKTextExtractionItem *item, IncludeRects includeRects)
 {
+    auto addRectIfNeeded = makeScopeExit([&] {
+        if (includeRects == IncludeRects::No)
+            return;
+        auto rect = item.rectInWebView;
+        stream << " rect=["_s << static_cast<int>(rect.origin.x) << ',' << static_cast<int>(rect.origin.y) << ',' << static_cast<int>(rect.size.width) << ',' << static_cast<int>(rect.size.height) << ']';
```

Comment:
Optional nit: maybe you could `auto rect = roundedIntRect(item.rectInWebView);` and then avoid the bunch of `static_cast`s in the next line?

Reply:
Nice! Much cleaner.

Reply:
Hm... iOS WKTR doesn't link WebCore, so this won't work. I think it's probably cleaner to abstract this out into a helper method to unduplicated some of these calls.